script/deploy/base/BaseDeploy.s.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.25;
+
+import {Script} from "forge-std/Script.sol";
+
+import {Create3, ICreateX} from "@symbioticfi/core/script/utils/Create3.sol";
+import {Logs} from "@symbioticfi/core/script/utils/Logs.sol";
+
+contract BaseDeployScript is Script, Create3 {
+    /**
+     * @notice Deploys a contract using Create3 and calls an initialization function
+     * @param deployer The address of the deployer
+     * @param salt An 11-byte salt value for deterministic address generation
+     * @param initCode The contract bytecode to deploy
+     * @param data The calldata for the initialization function call
+     */
+    function runDeploy(
```

Comment:
if the script is for external use are we sure that we want to leave only hardcoded createX deployment?